tests/integration/test_fim/test_files/test_restrict/test_restrict_valid.py

```diff
@@ -96,6 +96,7 @@ def get_configuration(request):
     return request.param
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Blocked for Issue #4077. When fixed this should be unblocked")

```

Comment:
```suggestion
@pytest.mark.skipif(sys.platform=='win32', reason="Blocked by #4077.")

``` 

Apply it in each skip.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4121/commits/0298b40cf597afaa64e9a7cb28bd02c3cce025b9